docs/communication/protocol/clear-player.md

```diff
@@ -0,0 +1,28 @@
+[<- Back](../index.md)
+
+# Clearing(Removing) Player (Player disconnect)
```

Comment:
Here, we need to wait a while before removing the player, as they might join right again with a new socket connection right?

Reply:
Add wait before removing the player - 603dcfafe557e403943dcafc2d76395524112d4a